modules/strawman-core/domain/service/sanitizeSnapshot.ts

```diff
@@ -0,0 +1,17 @@
+const SANITIZATIONS = [
+  {
+    match: /(?<!\\)`/g,
+    replace: "/`",
```

Comment:
```suggestion
    replace: "\\`",
```

Forward slash seems wrong to me here, since the inserted escape character should be a backslash. Or am I missing something?